src/scripts/burgerMenu.ts

```diff
@@ -0,0 +1,61 @@
+const Menu = () => {
+  const menu = document.querySelector(".menu") as HTMLDivElement;
```

Comment:
why are we doing it like this ? can't we use some package or UI library for this ?

Reply:
We could have avoided it, had the code been written using tailwind, in which case we could have used states and class names to achieve it. But bootstrap doesn't support this. Plus even we were to use a package we would still need to customize it via css to suit our needs.